services/upload/controller/controller.go

```diff
@@ -16,6 +16,10 @@ func SetupController(route *mux.Route) {
 	router.HandleFunc("/resume/", GetCurrentUserResume).Methods("GET")
 	router.HandleFunc("/resume/{id}/", GetUserResume).Methods("GET")
 
+	router.HandleFunc("/photo/upload/", GetUpdateUserPhoto).Methods("GET")
```

Comment:
```suggestion
	router.HandleFunc("/photo/", UpdateUserPhoto).Methods("PUT")
```
Note the change in handler function name.

Reply:
This has to be a GET request, because it's getting a URL from S3. Although I think the method name `GetUpdateUserPhoto` is confusing